docs/contributing/Local_Development.mdx

```diff
@@ -61,23 +61,6 @@ You can run `yarn test` in any package to run its tests.
 
 > [VS Code launch tasks](https://code.visualstudio.com/docs/editor/tasks) tasks are provided that allow [visual debugging](https://code.visualstudio.com/docs/editor/debugging) tests.
 
-#### Code Coverage
-
-We aim for 100% code coverage in all PRs when possible, except in the `website/` package.
-Coverage reports are generated locally whenever `yarn test` is run.
-
-The `codecov` bot should also comment on your PR with the percentage, as well as links to the line-by-line coverage of each file touched by your PR.
-
-#### Granular Unit Tests
-
-Most tests in most packages are set up as small, self-contained unit tests.
-We generally prefer that to keep tests and their failure reports granular.
-
-For rule tests we recommend, when reasonable:
-
-- Including both `valid` and `invalid` code changes in most PRs that affect rule behavior
-- Limiting to one error per `invalid` case
-
 ### Type Checking
```

Comment:
Hmm. should this section move too? 🤔

Reply:
I don't think so - I see this page as the _"here are the raw commands you'll use"_.